### PR TITLE
Improve `ftl status` output readability 

### PR DESCRIPTION
**Pull request by KendallWeihe**
Added JSON formatting w/ indents & line breaks for the ftl status output -- previously was a giant JSON all on one line

### CODE REVIEW

**Comment by KendallWeihe** [comment]
Accidentally closed [prior PR](https://github.com/TBD54566975/ftl/pull/454) b/c renamed branch

This is the active PR now

**Comment by KendallWeihe** [comment]
Also formatted the `ftl ps --json` per @wesbillman's [comment](https://github.com/TBD54566975/ftl/pull/454#issuecomment-1749411093)

File: cmd/ftl/cmd_ps.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ func (s *psCmd) Run(ctx context.Context, client ftlv1connect.ControllerServiceCl
 		return errors.WithStack(err)
 	}
 	if s.JSON {
-		marshaller := jsonpb.Marshaler{}
+		marshaller := jsonpb.Marshaler{Indent: "  "}
```
Comment (alecthomas) [inline cmd/ftl/cmd_ps.go:30]: This one makes sense, but the other two shouldn't be indented, as they're part of a columnar output and indenting them will just corrupt the columns.

**Review by alecthomas — CHANGES_REQUESTED** [review]

File: cmd/ftl/cmd_ps.go (hunk 1 above)
Comment (KendallWeihe) [reply cmd/ftl/cmd_ps.go:30]: ah cool, fixed

**Review by alecthomas — APPROVED** [review]
Awesome, thanks Kendall!

**Pull request merged**